### PR TITLE
chore: Added stale bot config

### PR DESCRIPTION
**Pull request by prilutskiy**

### CODE REVIEW

**Comment by travis-tf** [comment]
*[BOT]* Preview available: https://embed.typeform.com/073a34ffc9b59357a35d189f4a0f73bd37a01110/embed.js

**Comment by mathio** [comment]
Since we are not notified when someone posts new issue (at least I am not, even though I have the repo starred and watched) I would wait with this until we have a notifications for new issues in our slack channel. Otherwise we might close an issue before we have a chance to see it.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```
Comment (mathio) [inline .github/stale.yml:4]: I would go with lower setting, I think 7 days of inactivity should be enough.

**Comment by lfons** [comment]
Is there a way to test it?

**Comment by prilutskiy** [comment]
@lfons only after it's merged

File: .github/stale.yml (hunk 1 above)
Comment (prilutskiy) [reply .github/stale.yml]: Done

**Pull request closed without merging**